docs/README.md

```diff
@@ -30,6 +30,18 @@ For configuration examples for the caddy plugin look at our [examples section](/
 ### Location
 The TXT record must be accessible under the subdomain "\_redirect".
 
+### URLs
+* Please encode all URLs in the TXT record since the parser expects encoded URLs. In some cases, unencoded characters may break your configuration.
+* Most notably, ";" should be escaped as "%3B" in a URL since the parser will split the URL on the unencoded semicolon.
+
+Examples: <br />
+; -> %3B <br />
+/ -> %2F <br />
+? -> %3F <br />
+= -> %3D <br />
```

Comment:
I may be wrong on that one but I think you can add two spaces at the end of a line to instruct markdown to add a linebreak. Allows the file to still be readable in text mode.

Reply:
Thanks - that works